rspace/src/main/scala/coop/rchain/rspace/nextgenrspace/history/AbstractLMDBStore.scala

```diff
@@ -0,0 +1,64 @@
+package coop.rchain.rspace.nextgenrspace.history
+
+import java.nio.ByteBuffer
+
+import cats.effect.Sync
+import org.lmdbjava.{Dbi, Env, Txn}
+
+import scala.util.control.NonFatal
+
+abstract class AbstractLMDBStore[F[_]: Sync] {
```

Comment:
We should probably figure out a way to share common LMDB code between block-storage and rspace:
https://github.com/rchain/rchain/pull/2349/files#diff-6a230b89a68797209f8dfa4e90401807R15